src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div>
+    <div>
+      <label>{{selectLabel}}<span v-if=isRequired>*</span></label>
+    </div>
+    <div>
+      <select
+        v-bind:aria-label="selectLabel"
+        v-bind:required="isRequired ? true : false"
+      >
+        <option disabled selected>Choose Option</option>
+        <BSelectOption
```

Comment:
Do we need to have a component for this? It just seems like an option and doesn't really need to be a component.

Reply:
It felt like the right thing to do here.